specifications/KeyValueStore/KVsnap.tla

```diff
@@ -12,9 +12,12 @@ CONSTANTS   Key,            \* The set of all keys.
 \* Instantiating ClientCentric enables us to check transaction isolation guarantees this model satisfies
 \* https://muratbuffalo.blogspot.com/2022/07/automated-validation-of-state-based.html         
 CC == INSTANCE ClientCentric WITH Keys <- Key, Values <- TxId \union {NoVal}          
-
+    
+\* for instantiating the ClientCentric module
 wOp(k,v) == CC!w(k,v)
 rOp(k,v) == CC!r(k,v)
+InitialState == [k \in Key |-> NoVal]   
+SetToSeq(S) == CHOOSE f \in [1..Cardinality(S) -> S] : IsInjective(f)
```

Comment:
Consider using https://github.com/tlaplus/CommunityModules/blob/f070b68b02cf5b6d085797402374d85f739c167e/modules/SequencesExt.tla#L22-L27

Reply:
Will keep this in mind for future models. I don't want to change manifest.json and depend on SequencesExt now.

Reply:
Automation gets in the way. :-(